crates/orchestrator/src/tests/common/mod.rs

```diff
@@ -74,3 +77,10 @@ pub fn custom_job_item(default_job_item: JobItem, #[default(String::from("0"))]
 
     job_item
 }
+
+pub async fn drop_database() -> color_eyre::Result<()> {
+    let db_client: Client = MongoDb::new(MongoDbConfig::new_from_env()).await.client();
+    // dropping `jobs` collection.
+    db_client.database("orchestrator").collection::<JobItem>("jobs").drop(None).await?;
```

Comment:
Careful that you'll need to add every future collection in here as well. It might be better to just drop every collection under the `orchestrator` DB.

Reply:
Valid, changed to dropping all the collections for now.
+ Added comment to how to change to specific collection.